### PR TITLE
Smashed some typos in a post

### PR DESCRIPTION
**Pull request by johanbove**
You 🎸  Wes! But I found some typos in this - older - article which I wanted to correct and help you out a little with keeping up your immaculate image. Hope you don't mind 🍻

### CODE REVIEW

File: src/posts/Setting Default Values with JavaScripts Destructuring/Setting Default Values with JavaScripts Destructuring.mdx [hunk 1]
```diff
@@ -70,8 +70,7 @@ console.log(dogName) // what will it be? 0!
 
 <h3>Combining with Destructuring Renaming</h3>
 
-In my last post we learned that we can destrucutre and rename varaibles at the same time with something like this:
```
Comment (johanbove) [inline src/posts/Setting Default Values with JavaScripts Destructuring/Setting Default Values with JavaScripts Destructuring.mdx:73]: Warning. Here be typos!

File: src/posts/Setting Default Values with JavaScripts Destructuring/Setting Default Values with JavaScripts Destructuring.mdx [hunk 2]
```diff
@@ -38,7 +38,7 @@ console.log(speed); // 150 - comes from settings object
 console.log(width); // 500 - fallback to default
 ```
 
-Now if the `speed or width properties don't exist on our settings object, they fallback to 750 and 500` respectively.
+Now if the `speed` or `width` properties don't exist on our `settings` object, they fallback to `750` and `500` respectively.
```
Comment (johanbove) [inline src/posts/Setting Default Values with JavaScripts Destructuring/Setting Default Values with JavaScripts Destructuring.mdx:41]: Thought it looks better to format variables and keys as inline code

File: src/posts/Setting Default Values with JavaScripts Destructuring/Setting Default Values with JavaScripts Destructuring.mdx [hunk 3]
```diff
@@ -70,8 +70,7 @@ console.log(dogName) // what will it be? 0!
 
 <h3>Combining with Destructuring Renaming</h3>
 
-In my last post we learned that we can destrucutre and rename varaibles at the same time with something like this:
-
```
Comment (johanbove) [inline src/posts/Setting Default Values with JavaScripts Destructuring/Setting Default Values with JavaScripts Destructuring.mdx:74]: Why the extra new lines?

**Comment by wesbos** [comment]
thanks!

**Pull request merged**